### PR TITLE
OSDOCS#5326: Added warning regarding rollbacks specific to TALM/SNO

### PR DESCRIPTION
**Pull request by obrown1205**
Version(s):
4.11+

Issue:
https://issues.redhat.com/browse/OSDOCS-5326

Link to docs preview:
https://59332--docspreview.netlify.app/openshift-enterprise/latest/scalability_and_performance/cnf-talm-for-cluster-upgrades.html#talo-backup-recovery_cnf-topology-aware-lifecycle-manager

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by mjpytlak — COMMENTED** [review]
@obrown1205 peer review lgtm.

File: modules/cnf-topology-aware-lifecycle-manager-backup-recovery.adoc [hunk 1]
```diff
@@ -9,6 +9,12 @@
 If an upgrade of a cluster fails, you can manually log in to the cluster and use the backup to return the cluster to its preupgrade state. There are two stages:
 
 Rollback:: If the attempted upgrade included a change to the platform OS deployment, you must roll back to the previous version before running the recovery script.
+
+[IMPORTANT]
```
Comment (mjpytlak) [inline modules/cnf-topology-aware-lifecycle-manager-backup-recovery.adoc:13]: @obrown1205  I am wondering if this admonition would be better placed after `Recovery`. IMO it unnecessarily breaks up your description list.

Comment (obrown1205) [reply modules/cnf-topology-aware-lifecycle-manager-backup-recovery.adoc:13]: Fair point :) updated with the latter suggestion. TY!

File: modules/cnf-topology-aware-lifecycle-manager-backup-recovery.adoc [hunk 2]
```diff
@@ -9,6 +9,12 @@
...
+====
+A rollback is only applicable to upgrades from TALM and SNO. This process does not apply to rollbacks from any other upgrade type.
```
Comment (stevsmit) [inline modules/cnf-topology-aware-lifecycle-manager-backup-recovery.adoc:15]: SNO must be spelled out, as per https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#single-node-openshift.

**Comment by stevsmit** [comment]
1 nit. Otherwise, LGTM

File: modules/cnf-topology-aware-lifecycle-manager-backup-recovery.adoc (hunk 2 above)
Comment (obrown1205) [reply modules/cnf-topology-aware-lifecycle-manager-backup-recovery.adoc]: good call :) tysm

**Comment by stevsmit** [comment]
lgtm

**Pull request merged**